support/util/util.go

```diff
@@ -188,3 +189,59 @@ func HashStruct(o interface{}) string {
 	intHash := hash.Sum32()
 	return fmt.Sprintf("%08x", intHash)
 }
+
+// ConvertRegistryOverridesToCommandLineFlag converts a map of registry sources and their mirrors into a string
+func ConvertRegistryOverridesToCommandLineFlag(registryOverrides map[string]string) string {
+	var commandLineFlagArray []string
+	for registrySource, registryReplacement := range registryOverrides {
+		commandLineFlagArray = append(commandLineFlagArray, fmt.Sprintf("%s=%s", registrySource, registryReplacement))
+	}
+	if len(commandLineFlagArray) > 0 {
+		sort.Strings(commandLineFlagArray)
+		return strings.Join(commandLineFlagArray, ",")
+	}
+	// this is the equivalent of null on a StringToString command line variable.
+	return "="
+}
+
+// ConvertImageRegistryOverridesToCommandLineFlag converts a map of image registry sources and their mirrors into a string
+func ConvertImageRegistryOverridesToCommandLineFlag(registryOverrides map[string][]string) string {
```

Comment:
nit: s/ConvertImageRegistryOverridesToCommandLineFlag/ConvertOpenShiftImageRegistryOverridesToCommandLineFlag/
(otherwise it's harder to distinguish between the 2)